### PR TITLE
Fix webpack usage to generate a sourcemap correctly

### PR DESCRIPTION
**Pull request by hikalium**
Webpack was not properly used so the generated source maps were for the js files generated by tsc, not the original ts file. Fix it by generating the correct source map with webpack by calling tsc from it, not from the Makefile.

Also add a CI recipe to make sure the typescript sources have no trivial errors by running build and lint.

### CODE REVIEW

**Comment by hikalium** [comment]
This change is ready for review. Please check it out!

**Comment by hikalium** [comment]
Note: Generating .js files for GitHub pages via GitHub actions will come in the future. For now, let me allow including the generated code to make it easy...

File: docs/servo_power_monitor/src/index.ts [hunk 1]
```diff
@@ -638,5 +638,6 @@ function setupDataLoad() {
   dropZone.innerText = 'Drop .json here';
   dropZone.addEventListener('dragover', handleDragOver, false);
   dropZone.addEventListener('drop', handleFileSelect, false);
+  console.log("hello!!!!!!!");
```
Comment (mhiramat) [inline docs/servo_power_monitor/src/index.ts:641]: Is this a debug message?

Comment (hikalium) [reply docs/servo_power_monitor/src/index.ts:641]: aaaahhhh, good catch... let me remove that...

File: docs/servo_power_monitor/src/index.d.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export {};
```
Comment (hikalium) [inline docs/servo_power_monitor/src/index.d.ts:1]: Note: automatically generated by webpack

File: docs/servo_power_monitor/src/index.ts (hunk 1 above)
Comment (hikalium) [reply docs/servo_power_monitor/src/index.ts]: done! Please review again ;)

**Review by mhiramat — APPROVED** [review]
LGTM!

**Pull request merged**